app/component/itinerary/navigator/NaviCardExtension.js

```diff
@@ -129,18 +131,66 @@ const NaviCardExtension = ({ legType, leg, nextLeg }, { config }) => {
       </div>
     );
   }
+  if (legType === LEGTYPE.MOVE && nextLeg) {
+    const { headsign, route, start } = nextLeg;
```

Comment:
This crashes or does something funny if nextLeg is not transit. Please change to  nextLeg?.transitLeg